util/types/bit.go

```diff
@@ -106,15 +106,15 @@ func ParseBit(s string, width int) (Bit, error) {
 	return Bit{Value: n, Width: width}, nil
 }
 
-// ParseBitValue parses the binary string for bit type into uint64.
-func ParseBitValue(s string, width int) (uint64, error) {
+// ParseStringToBitValue parses the binary string for bit type into uint64.
```

Comment:
substitute "binary string" with "string" usually written as "s/binary string/string" for short.